src/pages/CardCreate/Form/Number/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { forwardRef, useImperativeHandle, useRef } from 'react'
+import { FormInputProps } from '..'
+import FormArea from '../../../../components/Form/FormArea'
+import FormInput, {
+  FormInputBox,
+  FormInputElementRef,
+  InputDividerText,
+} from '../../../../components/Form/FormInput'
+import { useFormChangedDispatch } from '../../../../context/Form/hooks'
+import { CREATE_EXPIRE_MONTH } from '../constants/id'
+
+type CardNumberHandle = {
+  cardNumber1: () => string
+  cardNumber2: () => string
+  cardNumber3: () => string
+  cardNumber4: () => string
+}
+
+const CardNumberInput = forwardRef<CardNumberHandle, FormInputProps>(
+  (props, ref) => {
+    const inputRef1 = useRef<FormInputElementRef | null>(null)
+    const inputRef2 = useRef<FormInputElementRef | null>(null)
+    const inputRef3 = useRef<FormInputElementRef | null>(null)
+    const inputRef4 = useRef<FormInputElementRef | null>(null)
+    const dispatch = useFormChangedDispatch()
+
+    useImperativeHandle(ref, () => ({
+      cardNumber1() {
+        return inputRef1.current?.value() ?? ''
+      },
+      cardNumber2() {
+        return inputRef2.current?.value() ?? ''
+      },
+      cardNumber3() {
+        return inputRef3.current?.value() ?? ''
+      },
+      cardNumber4() {
+        return inputRef4.current?.value() ?? ''
+      },
```

Comment:
```suggestion
cardNums() {
        return {
          cardInput1: inputRef1.current?.value() ?? '', 
          cardInput2: inputRef2.current?.value() ?? '', 
          cardInput3: inputRef3.current?.value() ?? '', 
          cardInput4: inputRef2.current?.value() ?? '', 
        }
    },
```
객체로 묶어서 관리해도 괜찮지 않을까요? 값들이 너무 나눠져 있다보니 로직을 작성할때 반복적인 부분이 많은것 같습니다

Reply:
이렇게 관리하는게 확실히 더 좋아보입니다 ㅎㅎ
감사합니다!